.github/workflows/release-ci.yml

```diff
@@ -0,0 +1,45 @@
+name: "⚙️ Monitor Jenkins Release CI"
+on: 
+  schedule:
+    # Run every hour
+    - cron: "21 * * * *"
+
+  workflow_dispatch:   
+
+permissions:
+  contents: write
+  pull-requests: none 
+  issues: write
+  packages: none
+
+jobs:
+  security-scoring:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Jenkins Alert and Reporting
+        uses: UlisesGascon/jenkins-status-alerts-and-reporting@v1.3.0
+        id: jenkins-status-alerts-and-reporting
+        with:
+          database: monitor-release/database.json
+          jenkins-domain: 'ci.nodejs.org'
+          jenkins-username: ${{ secrets.JENKINS_RELEASE_USERNAME }}
+          jenkins-token: ${{ secrets.JENKINS_RELEASE_TOKEN }}
+          # Issues
+          generate-issue: true
+          issue-assignees: 'UlisesGascon'
```

Comment:
@richardlau not sure if makes sense to assign the issues to me, as I can't access to that Jenkins.

Reply:
I'm hoping that won't be the case for long 🙂.

Reply:
What level of access does the token give to the release jenkins instance? Just want to understand what the potential security impact might be.

Reply:
@mhdawson AFAIK we currently do not have a token with access to the release Jenkins instance.

Reply:
@richardlau right, but I think this would require one. I'm trying to understand what the risk of having a token with access to the release Jenkins in a public GitHub repo is.

Reply:
I think that we can limit the surface if we use read-only tokens. Not sure how much granularity Jenkins offers currently for this.

Reply:
Depending on how we can limit the tokens we might also want to have the actions run in a private repo?